### DISCUSSION TITLE
Yank to system clipboard on Wayland

### DISCUSSION THREAD

**Discussion by glow12121** [helix-editor/helix#6217] [Q&A] [answered]
Is there an option in Helix to yank and paste to and from the system clipboard? Specifically, I am using Wayland, Fedora's upcoming Sway spin.

I would like this to be default behavior, similar to the one `set clipboard+=unnamedplus` provides in Neovim.

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/wiki/Troubleshooting#copypaste-fromto-system-clipboard-not-working

**Reply by glow12121** [reply]
Is it possible, though, to make <key>space</key>+<key>y</key> and space+p the default actions? So that they replace <key>y</key> and <key>p</key>?

**Reply by kirawi** [reply]
Yes, you can rebind it. https://docs.helix-editor.com/remapping.html

**Reply by glow12121** [reply]
Here are the config lines for such a remapping: https://issuehint.com/issue/helix-editor/helix/2038
